torchao/prototype/quantized_training/bitnet.py

```diff
@@ -232,10 +235,22 @@ def backward(ctx, grad_output):
         return grad_input, grad_weight, grad_bias
 
 
-def bitnet_training():
-    return _get_linear_subclass_inserter(
-        BitNetTrainingLinearWeight, allow_requires_grad=True
-    )
+class BitNetTrainingConfig(AOBaseConfig):
+    pass
+
+
+# for bc
+bitnet_training = BitNetTrainingConfig
```

Comment:
ditto on bc break here

Reply:
I'd actually prefer to keep it, see previous PR for some additional context